src/calculateRentalCost.js

```diff
@@ -27,8 +27,16 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
-}
+const calculateRentalCost = days => {
+  let res = 40 * days;
```

Comment:
Let's create more meaningful name for variables:
```suggestion
  let totalCost = 40 * days;
```